### PR TITLE
[PXN-3446] Adjust text in consumer credits 

### PR DESCRIPTION
**Pull request by samantaalbanez**
## Motivación y Contexto

Text adjustment when there are interest-free installments

## Descripción

According to the new layout proposed by the UX team, it was suggested to change the text in front of the installment amount when there are interest-free installments. This change will be necessary to give more visibility to consumer credits.

## Cómo probarlo
When entering the payment screen, selecting the credit card option or customer credits with interest-free installments

## Screenshots
| Before | After |
|-------|---------|
Captura de Tela 2022-03-25 às 21 04 18|Captura de Tela 2022-03-25 às 21 01 47

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## Unreleased
+_XX_03_2022_
```
Comment (jvillada31) [inline CHANGELOG.md:2]:
```suggestion
_31_03_2022_
```

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/viewmodel/CreditCardDescriptorModel.java [hunk 2]
```diff
@@ -97,8 +107,13 @@ private void updateInstallmentsInfo(@NonNull final SpannableStringBuilder spanna
         if (payerCostSelected != PayerCost.NO_SELECTED) {
             return;
         }
+        final Text text = (isConsumerCredits && interestFree != null) ?
+                new Text(context.getString(R.string.px_customer_credits_installment_info),
+                        installmentsRightHeader.getBackgroundColor(),
+                        installmentsRightHeader.getTextColor())
+                    : installmentsRightHeader;
```
Comment (jvillada31) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/viewmodel/CreditCardDescriptorModel.java:114]: @samantaalbanez I know that we are only using this code here but I think is more clear to extract this into a function that validates `isConsumerCredits` && `interestFree` and returns either the `Text` from the `String` resource or the `Text` from `installmentsRightHeader`

File: px-checkout/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -257,4 +257,7 @@
     <string name="px_connectivity_error">No hay conexión. Intenta pagar más tarde.</string>
     <string name="px_snackbar_error_action">Volver</string>
 
+    <!-- Customer Credits -->
+    <string name="px_customer_credits_installment_info">"Condición especial para cuotas"</string>
```
Comment (jvillada31) [inline px-checkout/src/main/res/values/strings.xml:261]: Any special need for double quotes here and in the other String resources?

"Condición especial para cuotas" vs Condición especial para cuotas

**Review by jvillada31 — DISMISSED** [review]
LG, left a couple of comments / suggestions

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/viewmodel/CreditCardDescriptorModel.java (hunk 2 above)
Comment (samantaalbanez) [reply px-checkout/src/main/java/com/mercadopago/android/px/internal/viewmodel/CreditCardDescriptorModel.java:114]: adjusted !

**Comment by francofusaro** [comment]
Sami, that Text comes from backend, inside the OneTapItem.benefits node, is there a reason to not change it from backend (instead of doing it on front) when it's consumer credits? 🤔

**Comment by samantaalbanez** [comment]
PR closed, the backend will be changed

**Pull request closed without merging**